js/templates/modals/purchase/shipping.html

```diff
@@ -10,9 +10,9 @@ <h2 class="h4 required flexExpand"><%= ob.polyT('purchase.shippingTitle') %></h2
     <% ob.userAddresses.forEach((a, i) => { %>
     <option value="<%= i %>">
       <% let part1 = `${a.name}`;
+         part1 += a.company ? `, ${a.company},` : '';
```

Comment:
I'm wondering if since nearly all the address fields are optional, if it would be better to push any that exist into an array and then join it with a comma?